docs/content/preview/migrate/yb-voyager-cli.md

```diff
@@ -265,12 +266,28 @@ Example : `filename1:tablename1,filename2:tablename2[,...]`
 
 Default: '\t' (tab); can be changed to comma(,), pipe(|) or any other character.
 
-### –-has-header
+### --has-header
 
 This argument is to be specified only for CSV file type.
 
 Default: false; change to true if the CSV file contains column names as a header.
 
+**Note**: Boolean flags takes arguments in the format `--flag-name=[true|false]`, and not `--flag-name [true|false]`.
+
+### --file-opts
+
+Comma-separated string options for CSV file format. The options can include the following:
+
+1. `escape_char`: escape character
```

Comment:
do they need to be in a specific order?

Reply:
Any random order would work. Internally identifies and use it accordingly

Reply:
Use bullets then